test/test-_loadInIframe.js

```diff
@@ -126,6 +126,16 @@ buster.testCase('_loadInIframe',
       assert.equals(this.testObject.window, this._stubs.window);
       assert.equals(this.testObject.document, this._stubs.document);
       assert.equals(this.testObject.$, this._stubs.$);
+    },
+
+    'Throws an exception if $ is not defined on the iframe': function()
+    {
+      assert.exception(function()
+      {
+        this._stubs.$ = undefined;
```

Comment:
Ok, I'll move these stubbing operations out to a dedicated `setUp` method, with a proper `tearDown` as well. That should also stop the leaking to other tests.